source/options.html

```diff
@@ -4,6 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Chromatastic Options</title>
+        <link rel="stylesheet" href="styles/options.css">
```

Comment:
I couldn't get it to work with `import "tailwindcss/tailwind.css";` in `options.tsx`, but this works.